src/displayBackend/common/ConnectorBase.hpp

```diff
@@ -48,6 +51,74 @@ class ConnectorBase : public DisplayItf::Connector
 	 * @param height connector height as configured in XenStore
 	 */
 	ConnectorBase(domid_t domId, uint32_t width, uint32_t height);
+
+	/**
+	 * Queries connector's EDID
+	 * @param startDirectory grant table reference to the buffer start directory
+	 * @param size           buffer size
+	 */
+	size_t getEDID(grant_ref_t startDirectory, uint32_t size);
+
+private:
+
+	/* Refresh rate advertized via EDID detailed timings. */
+	const int EDID_REFRESH_RATE_HZ = 60;
+
+	/* Dots per inch advertized via EDID detailed timings. */
+	const int EDID_DPI = 100;
```

Comment:
This can be actually read from DRM / window manager - guest OS may use calculated DPI for proper representation of UI elements.

Reply:
will fix

Reply:
TODO